### ISSUE TITLE
Add support for multiple worker directories

### ISSUE THREAD

**Issue by florian-milky** [kettanaito/msw#1607]
### Scope

Improves an existing behavior

### Compatibility

- [ ] This is a breaking change

### Feature description

In my project I have several directories where I use msw (storybook and test for example).
At the moment I can only define one `workerDirectory`, so only one of my files get updated with new versions.
Suggestion: allow a `workerDirectories` option or allow `workerDirectory` to take an array of strings

**Comment by kettanaito** [comment]
Hi, @florian-milky. That's a great suggestion! I will try to add it to the upcoming major release. 

Would the following API work for you?

```json
{
  "msw": {
    "workerDirectory": ["./public", "./storybook/static", "foo/bar"]
  }
}
```

**Comment by kettanaito** [comment]
Thinking about this, I think the best strategy is to ship this incrementally _after_ the next release. This will have backward compatibility and will support a plain string as a value as well (I see no reason not to do that). May be one of the good issues to pick up after the release.

**Comment by florian-milky** [comment]
@kettanaito Sounds great! Thank you!

**Comment by zigang93** [comment]
It will be awesome for monorepo setup, can't wait for next major update release

**Comment by kettanaito** [comment]
## Released: v2.1.0 🎉

This has been released in v2.1.0!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v2.1.0)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/2.1.0)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
